lib/cluster.js

```diff
@@ -261,7 +261,7 @@ function masterInit() {
     assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
            'Bad cluster.schedulingPolicy: ' + schedulingPolicy);
 
-    var hasDebugArg = process.execArgv.some(function(argv) {
+    var hasDebugArg = process.execArgv.some(function hasDebugTest(argv) {
```

Comment:
might throw in the future tho. 
and about the es6 anon function idea, it contradicts what was said in the CTC meeting.